src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -118,5 +145,8 @@ public static ProviderCultureResult ParseCookieValue(string value)
 
             return new ProviderCultureResult(cultureName, uiCultureName);
         }
+
+        private static bool IsValidCulture(string cultureName) =>
+            CultureInfo.GetCultures(CultureTypes.SpecificCultures).Any(c => c.Name == cultureName);
```

Comment:
Here's what we do to actually parse the culture (https://github.com/aspnet/Localization/blob/dev/src/Microsoft.AspNetCore.Localization/RequestLocalizationMiddleware.cs#L156-L173). We could do the same check here to ascertain if the culture if something we'd eventually use. IMO, we could just move the test to the actual middleware and have it complain if we returned a `ProviderCultureResult` which we would not use.